solutions/client-landing-zone/setters.yaml

```diff
@@ -90,17 +90,23 @@ data:
   project-allowed-restrict-vpc-peering: |
     - under:projects/PROJECT_ID
   #
-  # nonp IP space
-  standard-nonp-cidr: 10.1.0.0/18
+  # nonp IP space, used by ComputeFirewallPolicyRule for network isolation
+  # this list must include all non protected cidr range including the ranges for GKE nodes, services and pods
```

Comment:
Maybe add an s to the first range - range > ranges.

Reply:
done in latest commit